ProjectBasic/app/src/main/java/com/example/moviedb/data/repository/UserRepository.kt

```diff
@@ -9,4 +10,19 @@ interface UserRepository {
     suspend fun getGenreList(): GenreResponse
     suspend fun getMovieList(page: Int, genre: Int): MovieResponse
     suspend fun getMovieDetail(movieId: Int, type: String): Movie
+
+    /**
+     * local movie db functions
+     */
+     fun getMovieListLocal(): LiveData<List<Movie>>
+
+    suspend fun getMovieLocal(id: Int): Movie?
+
+    suspend fun insertLocal(movie: Movie)
```

Comment:
tương tự các abstract dưới
```suggestion
    suspend fun insertMovieLocal(movie: Movie)
```